care/utils/tests/base.py

```
@@ -113,3 +113,16 @@ def attach_role_facility_organization_user(self, facility_organization, user, ro
         return FacilityOrganizationUser.objects.create(
             organization=facility_organization, user=user, role=role
         )
+
+    def create_service_request(self, patient, facility, encounter, **kwargs):
+        from care.emr.models import ServiceRequest
```

Ruff (0.12.2)

`import` should be at the top-level of a file

(PLC0415)